detekt-rules-ruleauthors/build.gradle.kts

```
@@ -2,12 +2,27 @@ plugins {
     id("module")
 }
 
+val generatedRuleauthorsConfig: Configuration by configurations.creating {
```

What do you think about naming these consuming configs the same in every module?
```suggestion
val generatedConfig: Configuration by configurations.creating {
```

Having the same name kind-of suggests that their names have to match, which isn't true. Having the same name also looks confusing at usage, compare:
`generatedRuleauthorsConfig(project(detektGenerator, "generatedRuleauthorsConfig"))`
vs
`generatedConfig(project(detektGenerator, "generatedRuleauthorsConfig"))`

---

Yes, I agree about the confusion and it was tempting to convert the string to the variable name although representing different things.
Let's name the consumer configuration as per your suggestion 👍